### PR TITLE
Skip tests for local storage refacto

### PR DESCRIPTION
**Pull request by ClementWalter**

### CODE REVIEW

File: crates/ef-testing/src/models/case.rs [hunk 1]
```diff
@@ -165,10 +169,15 @@ impl BlockchainTestCase {
...
+                let balance_diff = format!(
                     "balance mismatch for {:#20x}: expected {:#32x}, got {:#32x}",
                     address, expected_state.balance.0, actual
-                )));
+                );
+                diff.push(balance_diff);
+            }
+
+            if !diff.is_empty() {
+                return Err(RunnerError::Other(diff.join("\\n")));
```
Comment (greged93) [inline crates/ef-testing/src/models/case.rs:180]: I think we can remove this if it doesn't work

**Review by greged93 — COMMENTED** [review]
One comment

File: crates/ef-testing/src/models/case.rs (hunk 1 above)
Comment (ClementWalter) [reply crates/ef-testing/src/models/case.rs:180]: but I still need to input a string to `Other` and `diff` is a `Vec<String>`

Comment (greged93) [reply crates/ef-testing/src/models/case.rs:180]: Will be fixed in another PR

**Pull request merged**